server/jobs/storage_backend.go

```diff
@@ -19,7 +22,41 @@ type StorageBackend interface {
 	Read(key string) ([]string, error)
 
 	// Write logs to the storage backend
-	Write(key string, logs []string) (bool, error)
+	Write(key string, logs []string, fullRepoName string) (bool, error)
```

Comment:
Can you make a ticket to remove the feature flag and therefore also this from the interface?

Reply:
Done!